core/tests/protractor_desktop/skillEditor.js

```diff
@@ -167,6 +167,21 @@ describe('Skill Editor functionality', function() {
     await skillEditorPage.expectNumberOfMisconceptionsToBe(1);
   });
 
+  it(
+    'should show stale tab and unsaved changes status info modals',
+    async function() {
+      await skillEditorPage.openSkillEditorInNewTab(skillId);
```

Comment:
Can you close the newly-opened tab when you're done?

Reply:
Done.